packages/scxa-tsne-plot/html/Demo.js

```diff
@@ -132,7 +138,8 @@ class Demo extends React.Component {
     this.state = {
       selectedPlotType: plotTypeDropdown[0].plotType.toLowerCase(),
       geneId: ``,
-      selectedPlotOption: plotTypeDropdown[0].plotOptions[0],
+      selectedPlotOption: Object.values(plotTypeDropdown[0].plotOptions[0])[0],
+      selectedPlotOptionLabel: Object.keys(plotTypeDropdown[0].plotOptions[0])[0] + `: ` + Object.values(plotTypeDropdown[0].plotOptions[0])[0],
       selectedColourBy: ks[Math.round((ks.length -1) / 2)].toString(),
```

Comment:
is this calculation for the preferred `Options`  dropdown value while loading the page for the first time?

Reply:
I ran the branch in my local, I have a question for @lingyun1010 :

1. When I change to `tSNE` plotType - corresponding preferred value is : `perplexity:40`
![image](https://user-images.githubusercontent.com/15771356/121715871-46d3d800-cad7-11eb-8107-4ccea4569797.png)

2. When I change 'UMAP' plotType - corresponding preferred value is: `n_neighbors:5`

![image](https://user-images.githubusercontent.com/15771356/121716444-f14bfb00-cad7-11eb-9bd7-5d268a45874e.png)

is this the correct preferred value for the `UMAP` Options?

Reply:
I and @lingyun1010  had a meeting to discuss review comments, which got clarified.